docs/core/tools/troubleshoot-usage-issues.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Troubleshoot .NET Core tool usage issues
+description: Discover the common issues when running .NET Core tools and possible solutions.
+author: kdollard
+ms.date: 09/23/2019
+---
+
+# Troubleshoot .NET Core tool usage issues
+
+You might come across issues when trying to install or run a .NET Core tool, which can be a global tool or a local tool. This article describes the common root causes and some possible solutions.
+
+## Installed .NET Core tool fails to run
+
+When a .NET Core tool fails to run, most likely you ran into one of the following issues:
+
+* The executable file for the tool wasn't found.
+* The correct version of the .NET Core runtime wasn't found. 
+
+### Executable file not found
+
+If the executable file isn't found, you'll see a message similar to the following:
+
+```
+Could not execute because the specified command or file was not found.
+Possible reasons for this include:
+  * You misspelled a built-in dotnet command.
+  * You intended to execute a .NET Core program, but dotnet-xyz does not exist.
+  * You intended to run a global tool, but a dotnet-prefixed executable with this name could not be found on the PATH.
+```
+
+The name of the executable determines how you invoke the tool. The following table describes the format:
+
+| Executable name format  | Invocation format   |
+|-------------------------|---------------------|
+| `dotnet-<toolName>.exe` | `dotnet <toolName>` |
+| `<toolName>.exe`        | `<toolName>`        |
+
+* Global tools
+
+    Global tools can be installed in the default directory or in a specific location. The default directories are:
+
+    | OS          | Path                          |
+    |-------------|-------------------------------|
+    | Linux/macOS | `$HOME/.dotnet/tools`         |
+    | Windows     | `%USERPROFILE%\.dotnet\tools` |
+
+    If you're trying to run a global tool, check that the `PATH` environment variable on your machine contains the path where you installed the global tool and that the executable is in that path.
+
+    The .NET Core CLI tries to add the default locations to the PATH environment variable on its first usage. However, there are a couple of scenarios where the location might not be added to PATH automatically, such as:
+
+  * If you've set the `DOTNET_SKIP_FIRST_TIME_EXPERIENCE` environment variable.
+  * On macOS, if you've installed the .NET Core SDK using *.tar.gz* files and not *.pkg*.
+  * On Linux, you need to edit the shell environment file to configure the PATH.
```

Comment:
~~* On macOS, if you've installed the .NET Core SDK using *.tar.gz* files and not *.pkg*.~~
~~* On Linux, you need to edit the shell environment file to configure the PATH.~~

* You need to edit the shell environment file to configure the PATH
  - macOS after Catalina
  - macOS before Catalina,  installed the .NET Core SDK using *.tar.gz* files and not *.pkg*
  - Linux, installed the .NET Core SDK using *.tar.gz* files and not apt-get or rpm